tests/unit/dom-event-listeners-test.js

```diff
@@ -26,12 +26,6 @@ module('ember-lifeline/dom-event-listeners', function (hooks) {
           this._super(...arguments);
           testContext.componentInstance = this;
         },
-
-        destroy() {
-          this._super(...arguments);
-
-          runDisposables(this);
```

Comment:
I think this is good to remove, we should no longer _need_ to call `runDisposables` manually (since Ember / Glimmer do this for us).

We should add some tests that confirm things work fine when you **do** still call it manually though.

Reply:
Agree.

Reply:
To be clear, I can add separate tests that test the disposables APIs themselves.

Reply:
> I can add separate tests that test the disposables APIs themselves.

Yes, exactly. In that set of tests we **must** ensure that calling `runDestroyables` _from `willDestroy`_ does not break the system, it is a bit nuanced but IMHO quite important.

Reply:
Agree.

Reply:
We discussed offline and agreed to not add this, since effectively this is _not_ expected to work when this is invoked multiple times with the new APIs.

Reply:
The `runDisposables` function should effectively be a noop, as the system is now responsible for running the destructors.